src/index.js

```diff
@@ -124,16 +124,16 @@ export function CartProvider({
   const addItem = (item, quantity = 1) => {
     if (quantity <= 0) return;
     if (!item.id) throw new Error("You must provide an `id` for items");
-
+    if (!item.price) throw new Error("You must pass a `price` for new items");
```

Comment:
Hey @halafy-ds 

A price only needs to be passed if the item is not a `currentItem` as checked for in the statement below. Was there a specific use case that you needed to add this line for?

Reply:
Yes sorry i forgot. I’m mistaking. In this cas the binary operator should be replaced by logical operator. && instead of &